controller/src/routes/instances.rs

```diff
@@ -54,26 +88,31 @@ pub async fn post_instance(body: String) -> anyhow::Result<Json<Value>, ApiError
             // We spawn a thread to handle the request
             let mut client = Client::new().await?;
 
+            let instance_id = workload.instance_id.clone();
+
```

Comment:
I think at some point we use too many .clone()s. I don't know if this can have an impact on the system in the end, but at the beginning of the method, I think you can replace those lines:

```rs
let wr = json_request.as_ref().clone();
// Create a grpc workload object
let workload = Workload::from(wr.workload.clone());
```  
By this one :

```rs
let workload = Workload::from(json_request.0.workload);
```
It compiles for me, but I don't know if it works as expected. It's just a suggestion. :+1:

Reply:
Yeah, it seems to work correctly. Thank you.